rfc_bibtex/rfc_bibtex.py

```diff
@@ -35,8 +36,32 @@ def __init__(self, id_names=[], in_file_name=None, out_file_name=None):
         if in_file_name is not None:
             self._id_names += self._read_ids_from_file(in_file_name)
 
+    @staticmethod
+    def _rfc_key_function(name):
+        """Turn RFC32 into RFC00032 for sorting"""
+        try:
+            if name.startswith('RFC'):
+                return "RFC"+format( int(name[3:]), "05d")
+        except (ValueError,TypeError) as e:
+            pass
+        return name
+
+    latex_citation_re = re.compile(r"^\\citation\{((rfc.*)|(draft-.*))\}",re.I)
```

Comment:
Please put this as a constant at the top (you have examples there).

Reply:
Sure.